app/models/search_vector.rb

```diff
@@ -0,0 +1,8 @@
+class SearchVector < ActiveRecord::Base
+  belongs_to :discussion
+  self.table_name = 'discussion_search_vectors'
```

Comment:
If we end up going with this solution, we'll change this to just be 'search_vectors'